aptos-move/framework/aptos-framework/sources/TokenV1.move

```diff
@@ -708,14 +744,88 @@ module AptosFramework::TokenV1 {
         }
     }
 
+    /********************Token Capability*****************************/
+    struct TokenCapability has store {
```

Comment:
Why not copy the `Capability` module from the stdlib into the aptoslib and use it here instead of reimplementing all this? It has this and a bit more, and also does the naming right.